<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HoloMainWindow</class>
 <widget class="QMainWindow" name="HoloMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HoloMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="openGLWidget">
        <property name="minimumSize">
         <size>
          <width>480</width>
          <height>360</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
      </property>
      <item row="3" column="1">
       <widget class="QSlider" name="time_slider">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="tracking">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">    QPushButton:!hover{                                 
        border:none;                                    
        image:url(:/Image/icon/Image/icon/page4.png)    
}                                                       
    QPushButton:hover {                                 
        border:none;                                    
        image:url(:/Image/icon/Image/icon/page5.png);   
}                                                       
    QPushButton:pressed {                               
        border:none;                                    
        image:url(:/Image/icon/Image/icon/page4.png);   
} </string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="timestamp">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>00:00</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="stop_play">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:!hover{
	border:none;
	image:url(:/Image/icon/Image/icon/page1.png)
}
QPushButton:hover { 
	border:none;
	image:url(:/Image/icon/Image/icon/page0.png);
}
QPushButton:pressed { 
	border:none;
	image:url(:/Image/icon/Image/icon/page1.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="total_time">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>00:00</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QSlider" name="volume_slider">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maximum">
         <number>128</number>
        </property>
        <property name="value">
         <number>128</number>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="openFile"/>
    <addaction name="saveFile"/>
   </widget>
   <widget class="QMenu" name="tool_menu">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="netMode"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="tool_menu"/>
  </widget>
  <widget class="QDockWidget" name="dockTitleWidget">
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <action name="openFile">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="netMode">
   <property name="text">
    <string>网络模式</string>
   </property>
  </action>
  <action name="saveFile">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
